othello/templates/emails/tournament_notify.html

```diff
@@ -0,0 +1,21 @@
+<h4 style="color: #33a122">
+    Hello! This is an automated message from the Othello Tournament Server.
+</h4>
+<h5>
+    You are receiving this email because your AI was selected to participate in an Othello Tournament
+    <br>
+    If you receive this email multiple times this means the tournament has been rescheduled. The most recent email is always the most accurate
```

Comment:
```suggestion
    You are receiving this email because your AI was selected to participate in an Othello tournament.
    <br>
    If you receive this email multiple times, the tournament has been rescheduled. The most recent email is always the most accurate.
```